examples/source/waypoints/index.js

```diff
@@ -98,6 +98,8 @@ const WaypointsExample = ({
       <TourTracker
         tourers={decoratedTourers}
         selected={selected}
+        focusOnTourer={decoratedTourers[0].id}
```

Comment:
Ok cool, will add tests.

As for selected, I hadn't really thought of using selected as I had sort of separated selected and focused in my head. But it probably makes sense to use selected.

The one "issue" is that I will still need to add some sort of flag to say focus on the selected tourer? In most cases, we still want to focus on the route regardless of who is selected, so I will still need another flag to say "I want to focus in the selected tourer"